### PR TITLE
Add members 09-11-2020

### LINKED ISSUE #2176: REQUEST: New membership for vinny-sabatini
### GitHub Username

@vinny-sabatini

### Organization you are requesting membership in

@kubernetes-sigs

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @seanmalloy
- @Raffo

### List of contributions to the Kubernetes project
- PRs reviewed
  - https://github.com/kubernetes-sigs/external-dns/pull/1507
  - https://github.com/kubernetes-sigs/external-dns/pull/1436
- PRs authored
  - https://github.com/kubernetes-sigs/external-dns/pull/1752
  - https://github.com/kubernetes-sigs/external-dns/pull/1748
  - https://github.com/kubernetes-sigs/external-dns/pull/1736
  - https://github.com/kubernetes-sigs/external-dns/pull/1731
- Issues responded to
  - https://github.com/kubernetes-sigs/external-dns/issues/1558
  - https://github.com/kubernetes-sigs/external-dns/issues/1675
  - https://github.com/kubernetes-sigs/external-dns/issues/1754
- SIG projects I am involved with
  - [kubernetes-sigs/external-dns](https://github.com/kubernetes-sigs/external-dns)

### LINKED ISSUE #2184: REQUEST: New membership for @salaxander
### GitHub Username
@salaxander

### Organization you are requesting membership in
Kubernetes SIGs, Kubernetes

### Requirements
- [ X] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [ X] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [X ] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [X ] I am actively contributing to 1 or more Kubernetes subprojects
- [ X] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [ X] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @christopherhein 
- @parispittman
- @onlydole 
- @karenhchu 
- @jeremyrickard 

### List of contributions to the Kubernetes project
- PRs: 
https://github.com/kubernetes-sigs/multi-tenancy/pull/1099
https://github.com/kubernetes-sigs/multi-tenancy/pull/1065
https://github.com/kubernetes-sigs/multi-tenancy/pull/1005

- Issues responded to:
https://github.com/kubernetes-sigs/multi-tenancy/issues/1064
https://github.com/kubernetes-sigs/multi-tenancy/issues/1044

- SIGs:
 multi-tenancy

### PR DESCRIPTION
**Pull request by ameukam**
Fixes #2176
Fixes #2184

### CODE REVIEW

**Comment by ameukam** [comment]
/assign @mrbobbytables @nikhita 
cc @palnabarun @savitharaghunathan

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -865,6 +865,7 @@ members:
 - saiyam1814
 - saiyan86
 - sakshamsharma
+- salaxander
```
Comment (savitharaghunathan) [inline config/kubernetes/org.yaml:868]: Looks like @salaxander need another sponsor for joining K8s org. 

/hold

Comment (salaxander) [reply config/kubernetes/org.yaml:868]: added @karenhchu as a sponsor

Comment (salaxander) [reply config/kubernetes/org.yaml:868]: Also added @jeremyrickard

Comment (jeremyrickard) [reply config/kubernetes/org.yaml:868]: I approve :)

**Comment by mrbobbytables** [comment]
/hold cancel

Other sponsor has +1 \o/

**Comment by mrbobbytables** [comment]
oh woops, thought it had approve
/approve

**Pull request merged**